Website/sitecore/shell/client/Applications/WeBlog/Import.js

```diff
@@ -0,0 +1,138 @@
+                    }
+                    if (e.IsDone === false) {
+                        setTimeout(function (that) {
+                            that.updateProgress(that);
+                        }, 200, that);
+                    } else {
+                        that.ProgressBar1.set("maxValue", 1);
+                        that.ProgressBar1.set("value", 1);
+                        that.StatusMessage.set("text", "");
+
+                        if (e.Code === 200) {
+                            that.ProgressMessage.set("text", "Import complete");
+                        } else {
+                            that.ProgressMessage.set("text", e.Message);
+                        }
+                    }
+                }
+            });
+        },
+        tabManager: {
+            showSelectFile: function (that) {
+                that.tabManager.hideAllTabs(that);
+                that.TabControl1.viewModel.showTab("{BA5DAD8B-207D-4ADE-99AD-2B77B563BCA4}");
+            },
+            showUpload: function (that) {
+                that.tabManager.hideAllTabs(that);
+                that.TabControl1.viewModel.showTab("{D4FA7870-D836-4057-8B39-F2B3E2A7A31A}");
+            },
+            showImportLocation: function (that) {
+                that.tabManager.hideAllTabs(that);
+                that.TabControl1.viewModel.showTab("{21457E01-670E-4407-86F2-49E3BBD2153F}");
+            },
+            showImport: function (that) {
+                that.tabManager.hideAllTabs(that);
+                that.TabControl1.viewModel.showTab("{C5C44663-88A4-4311-A1B1-751FEDE2C1AB}");
+            },
+            hideAllTabs: function (that) {
+                $.each(that.TabControl1.viewModel.tabs(), function (e, d) {
+                    that.TabControl1.viewModel.hideTab(d);
+                }
+                );
+            }
+        },
+        addedFile: function () {
+            this.Uploader.viewModel.upload();
+            this.tabManager.showUpload(this);
+        },
+        acceptLocationButtonClick: function () {
+            this.updateDatasource(this, { ParentId: this.Location.viewModel.selectedItemId() });
+            this.txtbxBlogName.set("text", this.getData(this).BlogName);
+            this.tabManager.showImport(this);
+        },
+        importButtonClick: function () {
+            this.updateDatasource(this, {
+                BlogName: this.txtbxBlogName.get("text"),
+                BlogEmail: this.txtbxBlogEmail.get("text"),
+                ImportPosts: this.cbxPosts.viewModel.isChecked(),
+                ImportCategories: this.cbxCategories.viewModel.isChecked(),
+                ImportComments: this.cbxComments.viewModel.isChecked(),
+                ImportTags: this.cbxTags.viewModel.isChecked()
+            }),
+            $.ajax({
+                url: "/sitecore/api/ssc/Sitecore-Modules-WeBlog-Controllers/WordPressImport/1/ImportItems",
```

Comment:
We're using the endpoint URL in more than 1 location, so we should extract this as a variable at the top of the define scope and use the variable instead.